src/routes/drinks/+page.svelte

```diff
@@ -1,31 +1,14 @@
 <script lang="ts">
-  import PocketBase from "pocketbase";
+  import { pb } from "$lib/stores/authStore";
   import { onMount } from "svelte";
-  // prefixed with PUBLIC_ means that the variables can be read by the frontend. This is very unsafe, and is only temporary for testing !!
-  import {PUBLIC_PB_HOST, PUBLIC_PB_ADMIN_EMAIL, PUBLIC_PB_ADMIN_PASSWORD} from "$env/static/public";
 
-  async function getDrinks() {
-    // Init
-    const pb = new PocketBase(PUBLIC_PB_HOST);
-    await pb.admins.authWithPassword(PUBLIC_PB_ADMIN_EMAIL, PUBLIC_PB_ADMIN_PASSWORD);
+  let drinks: any[] = [];
```

Comment:
Kunne man declaret den riktige typen her i stedet for å bruke `any`?

Reply:
WIP, helt enig